cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -66,13 +66,16 @@
 public class IntegrationTestBase {
 
   @ClassRule
-  public static final SingletonExternalResource STANDALONE = new SingletonExternalResource(new StandaloneTester());
-
-  @ClassRule
   public static final TemporaryFolder TEMP_FOLDER = new TemporaryFolder();
 
+  public static final SingletonExternalResource STANDALONE = new SingletonExternalResource(new StandaloneTester());
+
   @Before
```

Comment:
The problem is that I don't want `getInstanceURI` to be `static` because we want our subclass to override how we obtain the instanceURI. This is so that we can read the instanceURI and accessToken from a file.